style.css

```diff
@@ -114,20 +114,27 @@ body {
   border-radius: 16px;
   background-color: #f7f7f9;
```

Comment:
[OPTIONAL]
- [ ] Kindly remove this `background-color: #f7f7f9;` from this section (`.work-section`) and add it to the `main`. That will applied the color to the `main` tag and make your code DRY :muscle: 
e.g.
```sh
main {
   background-color: #f7f7f9;
}
```
_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_